### PR TITLE
[11215] Adding user account to created EVSSClaims

### PR DESCRIPTION
**Pull request by bosawt**
## Description of change
This PR adds `UserAccount` to `EVSSClaims` records, when they are created

## Original issue(s)
department-of-veterans-affairs/vets-api#11215

### CODE REVIEW

**Review by kpethtel — APPROVED** [review]
Looks good. Bit of a pity we have to go through the same process in three locations. It would also be nice to have some sort of associated spec change, but afaict this isn't really well tested anywhere.

**Review by holdenhinkle — APPROVED** [review]

File: app/models/evss_claim.rb [hunk 1]
```diff
@@ -3,5 +3,21 @@
...
+  def self.claim_for_user_uuid(user_uuid)
+    return EVSSClaim.none unless user_uuid
+
+    where(user_uuid: user_uuid)
+  end
+
+  def self.claim_for_user_account(user_account)
+    return EVSSClaim.none unless user_account
+
+    where(user_account: user.user_account)
```
Comment (jerekshoe) [inline app/models/evss_claim.rb:21]: @bosawt Is this supposed to be:
```ruby
where(user_account: user_account)
```
We are starting to see this error in staging as of the last hour or so: http://sentry.vfs.va.gov/organizations/vsp/issues/173747

Comment (bosawt) [reply app/models/evss_claim.rb:21]: @jerekshoe I just merged a fix into staging now

Comment (jerekshoe) [reply app/models/evss_claim.rb:21]: Ok cool, thanks for letting me know!

**Pull request merged**